atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/OracleDbTableFactory.java

```diff
@@ -95,7 +97,7 @@ public DbWriteTable createWrite(TableReference tableRef, ConnectionSupplier conn
         TableValueStyle tableValueStyle = valueStyleCache.getTableType(conns, tableRef, config.metadataTable());
         switch (tableValueStyle) {
             case OVERFLOW:
-                return OracleOverflowWriteTable.create(config, conns, oracleTableNameGetter, tableRef);
+                return OracleOverflowWriteTable.create(config, conns, oracleTableNameGetter, oraclePrefixedTableNames, tableRef);
```

Comment:
doing above will also fix checkstyle  `Line is longer than 120 characters (found 129).`